### DISCUSSION TITLE
news/2021-09-29-user-forum-3

### DISCUSSION THREAD

**Discussion by giscus** [mhm-ufz/mhm-ufz.github.io#3] [website]
# news/2021-09-29-user-forum-3

The 3rd User-Forum on October 6th 2021 in GatherTown…

https://mhm-ufz.org/news/2021-09-29-user-forum-3

**Comment by pallavhydro** [comment]
So many new stuffs being released from mHM team.

We have a website. Plus now installation is a one-liner via conda. 

Olda's (Oldrich Rakovec) pre-processing tips are definitely going to be super useful to mHM-starters!
